Source/Basetypes/BigFloat.cs

```diff
@@ -157,6 +157,56 @@ public static BigFloat FromBigDec(BigDec v, int significandSize, int exponentSiz
       return new BigFloat(isNeg, sig, exp, sigSize, expSize);
     }
 
+    [Pure]
+    public static BigFloat FromBVString(String s)
+    {
+      /*
+       * String must be either of the format (fp (bv* *) (bv* *) (bv* *))
+       * or of the special value format: (_ val * *)
+       * Where * indicates an integer value
+       * and val is one of the special values: NaN, +oo, -oo, zero, -zero
+       */
+      BIM sig, exp;
+      int sigSize, expSize;
+      bool isNeg;
+
+      String[] args;
+
+      if (s.IndexOf("_") == 1)
```

Comment:
Sometimes you use `" "` and sometimes `' '` for strings. I think you mostly use `' '` and so maybe you should update this one.